pkg/db/db_mock.go

```diff
@@ -0,0 +1,22 @@
+package db
+
+import (
+	"context"
+
```

Comment:
Remove extra new line.

Reply:
I have my own rule 😎 

```
standard packages

external packages

internal packages
```

For example,

```
	"context"
	"io/ioutil"
	"os"
	"path/filepath"

	"github.com/stretchr/testify/assert"
	"github.com/stretchr/testify/require"
	"golang.org/x/xerrors"

	"github.com/aquasecurity/trivy-db/pkg/db"
	"github.com/aquasecurity/trivy/pkg/log"
```

What do you think of it?

Reply:
haha that's interesting. I think it's OK to leave it like this if you prefer.